src/server/HelloCodeaholics.Web/Controllers/PharmacyController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using HelloCodeaholics.Services.Application;
+using HelloCodeaholics.Core.Interfaces;
+using HelloCodeaholics.Core.Domain.Entities;
+
+namespace HelloCodeaholics.Web.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class PharmacyController : ControllerBase
+    {
+        private IPharmacyService _pharmacyService { get;set; }
+        public PharmacyController(IPharmacyService pharmacyService)
+        {
+            _pharmacyService = pharmacyService;
+        }
+
+        [HttpGet("{id}")]
+        public async Task<ActionResult<Pharmacy>> GetPharmacy(int id)
+        {
+            var response = await _pharmacyService.GetPharmacyById(id);
+            if (response is null)
+            {
+                return NotFound();
+            }
+            else
+            {
+                return Ok(response);
+            }
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<List<Pharmacy>>> GetPharmacies()
```

Comment:
Curious why you chose the name `GetPharmacies` for the controller method, and `GetPharmacyList` for the service method?

Reply:
Hmmm. No burning reason either way.  

It's easy enough to rename `GetPharmacyList` to `GetPharmacies` to provide some consistancy.

Reply:
Hi Shannon, this is Dery and I work alongside Adam and Jyoti. _(Just an FYI)_

I would argue towards naming the method GetPharmacyList instead of GetPharmacies as it seems to be more descriptive and less ambiguous. It will also help those who are not proficient in English to know exactly what that method is doing without having to drill into said method.

Reply:
Done.